README.md

```diff
@@ -201,5 +201,90 @@ to check that it hasn't crashed.
 
 Congratulations, you have just run your first GRChombo simulation!
 
+### Visualisation
+
+In this section, we will outline how to get set up with visualisation using
+Paraview on Fawcett. It is a bit fiddly, so we will go over the bare minimum
+to get up and running. For more detailed information on what Paraview can do,
+have a look at the
+[tutorial](https://www.paraview.org/Wiki/images/b/bc/ParaViewTutorial56.pdf)
+(or ask Amelia!). Adapted from instructions provided by Kacper Kornet.
+
+First, we need to set up ssh access to fawcett. Information on how to do this
+is given [here](https://www.maths.cam.ac.uk/computing/fawcett-ssh-access). We
+have sent this to you in advance, so you should have all of the steps set up
+prior to the tutorial.
+
+Second, we need you to download Paraview v5.6.0 onto your local machine i.e. the
+laptop/desktop you are using to access fawcett. This can be done
+[here](https://www.paraview.org/download/). Again, we will have asked you to
```

Comment:
It might be worth stressing that it is important that version 5.6.0 _specifically_ must be downloaded and not another version. 
```suggestion
Second, we need you to download Paraview v5.6.0 (note that is important that
you download this specific version and not a different one) onto your local machine i.e. the
laptop/desktop you are using to access fawcett. This can be done
[here](https://www.paraview.org/download/). Again, we will have asked you to
```